### PR TITLE
(WIP) Hanging Offsets

### PR DESCRIPTION
**Pull request by uSkizzik**
- Changes the default hanging offset to a new one
- [WIP] Adds offsets for vanilla entities via a map to allow mods to add their own entities if they care enough

### CODE REVIEW

File: src/main/java/subaraki/hangman/entity/HangEntityDummy.java [hunk 1]
```diff
@@ -102,10 +102,10 @@ public double getPassengersRidingOffset() {
         if (!this.getPassengers().isEmpty()) {
             Entity e = this.getPassengers().get(0);
             if (!(e instanceof Player))
-                return -e.getEyeHeight();
+                return -e.getEyeHeight() + 0.35;
         }
 
-        return -1.37D;
+        return -1.42D;
```
Comment (uSkizzik) [inline src/main/java/subaraki/hangman/entity/HangEntityDummy.java:108]: Looks better in my opinion.
Also, makes the noose look the same on both players and zombies.

File: build.gradle [hunk 2]
```diff
@@ -36,6 +36,11 @@ minecraft {
             }
         }
 
+        client2 {
```
Comment (uSkizzik) [inline build.gradle:39]: Another run configuration to run a second client for testing purposes.

**Review by ArtixAllMighty — CHANGES_REQUESTED** [review]
This PR has the build gradle changed.
Thus has nothing to do with the hanging offset, and should be either its won PR, or should be kept locally, for personal use, as I already have a second run set up from my IDE.

**Comment by ArtixAllMighty** [comment]
this feature has been implemented together with the white list

**Pull request closed without merging**